.travis.yml

```diff
@@ -50,6 +53,14 @@ install:
 script:
   - .ci/run_tests.sh
 
+jobs:
+  include:
+    - stage: deploy
+      if: tag =~ 0.
```

Comment:
@hutchic does this mean "only tags starting with `0.`"? If so, that should be good enough for our pre-1.0 tag use, as long as we're aware that those tags have this side-effect :+1:

Reply:
@hishamhm yes this will match `0.*`. We could make this more explicit so we don't accidentally build a release for accidentally pushed tags ie we could do `if: tag =~ ^release/0.` which would only match tags that are `release/0.*` ?

Reply:
Currently, the branch names follow the `release/` pattern but the release tags themselves are plain version numbers. I don't know if we can check both tag and branch, but if possible I guess that would be ideal.

Reply:
hmm maybe not then :/

I've seen issues where tag name == a branch name